framework/caching/CRedisCache.php

```diff
@@ -177,6 +177,17 @@ private function parseResponse()
 	}
 
 	/**
+	 * Counting amount of bytes in a string.
+	 *
+	 * @param string $str
+	 * @return int
+	 */
+	private function lenBytes($str)
```

Comment:
There's already such method in a helper. Could be used as `StringHelper::byteLength($string)`.